examples/fluid/dam_break_2phase_2d.jl

```diff
@@ -0,0 +1,95 @@
+# 2D dam break simulation with an air layer on top
+
+using TrixiParticles
+using OrdinaryDiffEq
+
+# Size parameters
+H = 0.6
+W = 2 * H
+
+gravity = 9.81
+tspan = (0.0, 2.0)
+
+# Resolution
```

Comment:
```suggestion
# ==========================================================================================
# ==== Resolution
```
Please follow the conventions of other example files.
For example, `gravity` and `tspan` should go below this under "Experiment Setup".